packages/react-components/react-dialog/stories/Dialog/DialogTriggerOutsideDialog.md

```diff
@@ -1,8 +1 @@
-There's absolutely no benefit in using `DialogTrigger` outside of `Dialog`, using a simple `Button` with `aria-expanded` properly set would be equivalent.
-
-Some disadvantages of not having the `DialogTrigger` inside the `Dialog`:
-
-1. `aria-expanded` should be manually configured
-2. `onOpenChange` callback won't fire `triggerClick` events, since there is not trigger to be clicked in context.
-
-> ⚠️ Do not forget to manually add `aria-expanded` attributes to ensure accessibility
+There's absolutely no benefit in using `DialogTrigger` outside of `Dialog`, just use controlled state with a button instead.
```

Comment:
@jurokapsiar you initially brought this up as a requirement which resulted in the DialogTrigger as a component. Has that requirement changed? 

If we are happy with this change, we should deprecate the `DialogTrigger` update all the examples to stop using them, I don't see why we would need a DialogTrigger at this point